modin/pandas/base.py

```diff
@@ -1711,7 +1711,7 @@ def _mad(self, axis, skipna, level):
         )
 
     @_inherit_docstrings(pandas.DataFrame.mask, apilink="pandas.DataFrame.mask")
-    def _mask(self, *args, **kwargs):
+    def _compat_mask(self, *args, **kwargs):
```

Comment:
why isn't this supposed to match the pandas api as just "mask"?

Reply:
@jbrockmendel -- Do you know how Pandas avoids namespace conflicts with the Numpy _mask attribute? We were getting this issue where np.ma.ravel was trying to run on the _mask method instead of the _mask attribute, and throwing an error as a result, which was breaking other functions (Matplotlib's scatter, for example). So we changed the _mask method name here, and that fixed the problem. But if there are better ways to do this (and stay closer to Pandas), that could definitely be the right call

Reply:
pd.DataFrame has `mask`, not `_mask`, so im not clear what the issue is

Reply:
@jbrockmendel this method exists for compatibility reasons. We have `mask` in the `_compat` module, which calls this `_mask`.

Reply:
I think current agreement is we're removing the compat thingy? So this should be reverted during that removal.